crm_compassion/__manifest__.py

```diff
@@ -62,6 +62,7 @@
         'views/demand_weekly_revision.xml',
         'views/hold_view.xml',
         'views/calendar_view.xml',
+        'views/email_view.xml',
```

Comment:
Naming the file `email_view` is confusing because the related object is not email but mail_message. So I would suggest that you name the file `mail_message_view.xml` instead. Other than that everything looks good!